modules/apps/client-extension/client-extension-type-item-selector-web/src/test/java/com/liferay/client/extension/type/item/selector/web/internal/item/selector/CETItemSelectorViewDescriptorTest.java

```diff
@@ -87,13 +104,46 @@ public void testGetSearchContainer() throws PortalException {
 			""
 		);
 
+		_testGetSearchContainer(
+			globalJSCET1, globalJSCET2,
+			ClientExtensionEntryConstants.TYPE_GLOBAL_JS);
+
+		ThemeCSSCET themeCSSCET1 = Mockito.mock(ThemeCSSCET.class);
+
+		Mockito.when(
+			themeCSSCET1.getScope()
+		).thenReturn(
+			"controlPanel"
+		);
+
+		ThemeCSSCET themeCSSCET2 = Mockito.mock(ThemeCSSCET.class);
+
+		Mockito.when(
+			themeCSSCET2.getScope()
+		).thenReturn(
+			""
+		);
+
+		_testGetSearchContainer(
+			themeCSSCET1, themeCSSCET2,
+			ClientExtensionEntryConstants.TYPE_THEME_CSS);
+	}
+
+	private void _testGetSearchContainer(CET cet1, CET cet2, String type)
+		throws PortalException {
+
+		Mockito.when(
+			_cetItemSelectorCriterion.getType()
+		).thenReturn(
+			type
+		);
+
 		Mockito.when(
 			_cetManager.getCETs(
-				Mockito.anyLong(), Mockito.any(),
-				Mockito.eq(ClientExtensionEntryConstants.TYPE_GLOBAL_JS),
+				Mockito.anyLong(), Mockito.any(), Mockito.eq(type),
 				Mockito.any(), Mockito.any())
 		).thenReturn(
-			Arrays.asList(globalJSCET1, globalJSCET2)
+			Arrays.asList(cet1, cet2)
```

Comment:
There is a ListUtil#fromArray in kernel module for this...Do you know if there is any liferay recommendation for this? :thinking: